text/assets/0225-config.yaml

```diff
@@ -0,0 +1,420 @@
+# include version specification in configuration files to help with parsing and schema evolution.
+scheme_version: 0.1
+sdk:
+  # Disable the SDK for all signals.
+  #
+  # Boolean value. If "true", a no-op SDK implementation will be used for all telemetry
+  # signals. Any other value or absence of the variable will have no effect and the SDK
+  # will remain enabled. This setting has no effect on propagators configured through
+  # the OTEL_PROPAGATORS variable.
+  #
+  # Environment variable: OTEL_SDK_DISABLED
+  disabled: false
+  # Configure resource attributes and resource detection for all signals.
+  resource:
+    # Key-value pairs to be used as resource attributes.
+    #
+    # Environment variable: OTEL_RESOURCE_ATTRIBUTES
+    attributes:
+      # Sets the value of the `service.name` resource attribute
+      #
+      # Environment variable: OTEL_SERVICE_NAME
+      service.name: !!str "unknown_service"
+  # Configure context propagators. Each propagator has a name used to configure it.
+  #
+  # Environment variable: OTEL_PROPAGATORS
+  propagators: [tracecontext, baggage, b3multi]
+  # Configure general attribute limits. See also sdk.tracer_provider.span_limits, sdk.logger_provider.log_record_limits.
+  attribute_limits:
+    # Set the max attribute value size.
+    #
+    # Environment variable: OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT
+    attribute_value_length_limit: 4096
+    # Set the max attribute count.
+    #
+    # Environment variable: OTEL_ATTRIBUTE_COUNT_LIMIT
+    attribute_count_limit: 128
+  # Configure the tracer provider.
+  tracer_provider:
+    # Span exporters. Each exporter key refers to the type of the exporter. Values configure the exporter. Exporters must be associated with a span processor.
+    exporters:
+      # Configure the otlp exporter.
+      otlp:
```

Comment:
Is this how the OTLP exporter would be selected, or would this configuration only happen if the OTLP exporter is selected how it currently is with entry points and such?

Reply:
On a related note, I am bit confused as to how 3rd party exporters would be configured with this method. Is the idea that users would just add a new section that the 3rd party exporter would search the configuration for?

Reply:
A similar question was discussed in this conversation: https://github.com/open-telemetry/oteps/pull/225#discussion_r1119292022

Please take a look

Reply:
@jeremydvoss There's a small section mentioning them here as well: https://github.com/open-telemetry/oteps/pull/225/files#diff-bd4bde8aa8b94a5c3332731193eb54f01b2000c861620e7ec6263d382a4b5979R32 

(We need further discussion to tune details once this hits the Specification, this will be definitely be supported, being a MUST)